src/store/actions.js

```diff
@@ -153,6 +153,21 @@ export default {
 
     return backendMethod(respondChallenge);
   },
+  async modifyNotifications({ state: { sdk }, getters: { activeNetwork } }, [ids, status, author]) {
```

Comment:
isn't author always the same? Proposing to do something like this:
```suggestion
  async modifyNotifications({ state: { sdk, account: { publicKey } }, getters: { activeNetwork } }, [ids, status]) {
```